wrappers/core18_test.go

```diff
@@ -52,6 +52,12 @@ func (s *servicesTestSuite) TestAddSnapServicesForSnapdOnCore(c *C) {
 	restore := release.MockOnClassic(false)
 	defer restore()
 
+	restore = release.MockReleaseInfo(&release.OS{ID: "ubuntu"})
+	defer restore()
+
+	// reset root dir
+	dirs.SetRootDir(s.tempdir)
```

Comment:
the suite's `SetUpTest` already does this, though

Reply:
The one in SetUpTest() will initialize the paths in dirs with host specific ones. I need to do it again fter mocking release info.